testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/LabUserHelper.java

```diff
@@ -40,10 +40,15 @@
 public class LabUserHelper {
 
     private static final Map<LabUserQuery, LabConfig> sLabConfigCache = new HashMap<>();
+    private volatile static ConfidentialClientHelper instance = LabAuthenticationHelper.getInstance();
 
-    static List<ConfigInfo> getConfigInfos(LabUserQuery query) {
+    public static void resetWithSecret(final String secret) {
```

Comment:
Does this actually reset the LabAuthenticationHelper instance? It seems like `getInstance(String keyVaultSecret)` only uses the secret provided if the instance is null, otherwise it simply returns the old instance back.

Reply:
Good catch.  We must not be using it elsewhere in the tests, otherwise my attempt would not have worked.

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1151/commits/8342a626a3aef4c3bbdd064b1956e925bdf6b61a